app/models/certificate.server.ts

```diff
@@ -41,6 +44,10 @@ export function updateCertificateById(
   });
 }
 
+export function deleteAllCertificatesByUsername(username: Certificate['username']) {
```

Comment:
Why do we need this? A cert will not become invalid just because we delete it from here, on the other hand we lose the ability to revoke it. `deleteCertificateById` was designed to delete outdated certificates that no longer work

Reply:
It was there previously, I simply used it for the test to work. Should I delete it?

Reply:
Interesting, it did show up as new code. Regardless, it's gone now